### ISSUE TITLE
Import tensorflow model from keras generated one

### ISSUE THREAD

**Issue by denisdnl** [dotnet/machinelearning#5505]
### System information

- **OS version/distro**: Win 10
- **.NET Version (eg., dotnet --info)**: 5 

### Issue

- **What did you do?** Tried to create in keras a simple NN(7 inputs, 10 outputs), freeze the model and import it in ML.Net
- **What happened?** Exception
- **What did you expect?** To be successfully imported 

### Source code / logs

Hi all,
I created a simple neural net using keras in python, and wanted to export it to ML.Net. First try was to save it in "tf" format, but as i saw, ML.Net needs a frozen model. So i tried to freeze it using this approach: [https://hellobird.tistory.com/399](https://hellobird.tistory.com/399), i have generated the .pb file, but when i try to **mlContext.Model.LoadTensorFlowModel(pathToModel);** an ** Tensorflow.TensorflowException: 'Could not find meta graph def matching supplied tags: { serve }. To inspect available tag-sets in the SavedModel, please use the SavedModel CLI: `saved_model_cli`' exception is thrown 
**

The python code is this attached 

[test.txt](https://github.com/dotnet/machinelearning/files/5577780/test.txt)
and the pb file(I changed it's extension so i could upload):
[model.txt](https://github.com/dotnet/machinelearning/files/5577783/model.txt)

I saved the pb, i imported it to VS and the c# code:

` 
  
             var mlContext = new MLContext();
            
            var tensorFlowModel = mlContext.Model.LoadTensorFlowModel(pathToModel);
            
            var pipeline = mlContext.Transforms.Concatenate("Features",
                    new[] { "Bathrooms", "SqftLiving", "SqftLot", "Floors", "YearBuild", "YearRenovated", "Price" })
                .Append(tensorFlowModel.ScoreTensorFlowModel("Prediction/Softmax", "Features"))
                .Append(mlContext.Transforms.CopyColumns("Scores", "Prediction/Softmax"));

            var dataView = mlContext.Data.LoadFromEnumerable(Enumerable.Empty<House>(), tensorFlowModel.GetModelSchema());
            var transformer = pipeline.Fit(dataView);`

But on the second line i got that error.

Is there any known issue about this or any sample code that shows how to export a model from python in ML.Net that works?

Thanks ^_^
